.cargo/config

```diff
@@ -10,3 +10,10 @@ rustflags = [
     # new in 1.60 and generates false positives
     "-Aclippy::only_used_in_recursion",
 ]
+
```

Comment:
For the commit message body, please write in complete, properly-formatted sentences. Describe the problem, how your patch addresses it, and what the impact is. See also the Linux kernel guide to patch descriptions: https://www.kernel.org/doc/html/v5.19/process/submitting-patches.html#describe-your-changes

Reply:
Nit: The commit message looks better now,  but can be edited for brevity / clarity. For example, the title can just be "Add vector state context switch support". The remaining lines can be more verbose with something like "This adds changes to allow...". "It also adds...". Note that lines should wrap at 72 characters (don't think we exceeded it, but just noting).